lib/util.c

```diff
@@ -77,6 +77,15 @@ duo_common_ini_handler(struct duo_config *cfg, const char *section,
                 free(buf);
                 return (0);
             }
+            while (p[strlen(p) - 1] == '\\') {
+                p[strlen(p) - 1] = ' ';
+                q = strtok(NULL, " ");
+		new_length = strlen(p) + strlen(q) + 1;
```

Comment:
I was bouncing between Notepad++ and vim. I fixed the indenting and checked for any other lines that were indented with tabs instead of spaces. They should all be spaces like the rest of the code now.